sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/inspections/UnusedImportInspection.kt

```diff
@@ -22,11 +22,17 @@ class UnusedImportInspection : LocalInspectionTool() {
     manager: InspectionManager,
     isOnTheFly: Boolean
   ): Array<ProblemDescriptor> {
-    val columnTypes = file.findChildrenOfType<SqlDelightJavaTypeName>()
-      .map { javaTypeName -> javaTypeName.text }
+    val sqlDelightColumnTypes = file.findChildrenOfType<SqlDelightColumnType>()
+    val columnTypes = sqlDelightColumnTypes.mapNotNull {
+      it.javaTypeName?.parameterizedJavaType?.firstChild?.text ?: it.javaTypeName?.text
```

Comment:
@AlecStrong `parameterizedJavaType?.firstChild` is it a bad idea?

Reply:
unfortunately, it doesn't return types after annotations and type parameters, so I added `columnType.findChildrenOfType<SqlDelightJavaTypeName>()`